### PR TITLE
fix(Storage): encouraging message for empty filtered lists

### PR DESCRIPTION
**Pull request by ariser**
An encouraging message for empty lists of degraded and out-of-space Groups and Nodes, with a CTA to switch to the All filter

Before:

After:

### CODE REVIEW

File: src/containers/Tenant/Diagnostics/Network/Network.scss [hunk 1]
```diff
@@ -111,7 +111,7 @@
         display: flex;
         gap: 12px;
 
-        margin-bottom: 16px;
+        margin: 0 16px 16px 0;
```
Comment (ariser) [inline src/containers/Tenant/Diagnostics/Network/Network.scss:114]: The right margin on empty lists was forced by the image, now that it became smaller, the layout squashed. This fixes it.

File: src/containers/Storage/StorageGroups/StorageGroups.tsx [hunk 2]
```diff
@@ -252,18 +254,42 @@ function StorageGroups({data, tableSettings, visibleEntities, nodes}: StorageGro
         },
     ];
 
+    const renderEmptyState = (message: string) => {
+        return (
+            <EmptyState
+                image={<div className="no-problem" />}
+                title={message}
+                description="Everyting is fine!"
```
Comment (Raubzeug) [inline src/containers/Storage/StorageGroups/StorageGroups.tsx:262]: Should we use here i18n?

File: src/containers/Storage/StorageGroups/StorageGroups.tsx [hunk 3]
```diff
@@ -272,7 +298,7 @@ function StorageGroups({data, tableSettings, visibleEntities, nodes}: StorageGro
             columns={columns}
             settings={tableSettings}
             initialSortOrder={setSortOrder(visibleEntities)}
-            emptyDataMessage={emptyMessage}
+            emptyDataMessage={'No such groups'}
```
Comment (Raubzeug) [inline src/containers/Storage/StorageGroups/StorageGroups.tsx:301]: i18n?

File: src/containers/Storage/StorageGroups/StorageGroups.tsx [hunk 4]
```diff
@@ -252,18 +254,42 @@ function StorageGroups({data, tableSettings, visibleEntities, nodes}: StorageGro
         },
     ];
 
+    const renderEmptyState = (message: string) => {
```
Comment (Raubzeug) [inline src/containers/Storage/StorageGroups/StorageGroups.tsx:257]: Lets move it to the separate component!

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/containers/Storage/EmptyFilter/EmptyFilter.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    showAll = i18n('default_show_all'),
+    onShowAll,
+}: EmptyFilterProps) => (
+    <EmptyState
+        image={<div className="no-problem" />}
+        title={title}
+        description={message}
+        actions={onShowAll && [
+            <Button
+                key="show-all"
```
[the remark this answers is not in the document]
Comment (ariser) [reply src/containers/Storage/EmptyFilter/EmptyFilter.tsx:26]: Is there a reason it should be here?

File: src/containers/App/App.scss [hunk 6]
```diff
@@ -130,13 +130,8 @@ body,
 }
 
 .no-problem {
-    min-width: 325px;
+    width: 200px;
```
Comment (Raubzeug) [inline src/containers/App/App.scss:133]: May be this class should be moved to Illustration component too?

File: src/containers/Storage/EmptyFilter/EmptyFilter.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Raubzeug) [reply src/containers/Storage/EmptyFilter/EmptyFilter.tsx:27]: Sorry, overthought a bit :)

**Review by Raubzeug — APPROVED** [review]

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**